### DISCUSSION TITLE
Some colors are different compared to vscode.

### DISCUSSION THREAD

**Discussion by Skyrocket1653** [olimorris/onedarkpro.nvim#233] [Q&A]
vscode:
![2024-03-31T21:59:15,900251922+02:00](https://github.com/olimorris/onedarkpro.nvim/assets/115411576/ad01405c-715d-49c7-8c3f-7d20327c1a2f)
neovim:
![2024-03-31T21:58:04,616447566+02:00](https://github.com/olimorris/onedarkpro.nvim/assets/115411576/8e99a63b-8b73-4b4c-bb34-f724feaf5c8d)

vscode:
![2024-03-31T21:59:40,489847410+02:00](https://github.com/olimorris/onedarkpro.nvim/assets/115411576/d7f37c78-b546-4e2f-b033-3821145c0465)

neovim:
![2024-03-31T21:59:59,737754713+02:00](https://github.com/olimorris/onedarkpro.nvim/assets/115411576/4f0a3df7-78f4-4ac9-9f48-a8ca01193943)

There are some differences for example, the TcpListener is yellow in vscode but is purple in neovim which leads to lines almost being completely purple. Functions are blue in vscode while in neovim they are yellow. After the use keyword in vscode it is usually yellow but in neovim it is red.

How can I make it like vscode?

**Comment by olimorris** [comment]
Pretty much as per: https://github.com/olimorris/onedarkpro.nvim?tab=readme-ov-file#question-faqs

A lot will depend on the Tree-sitter parser and if it has highlight groups defined for what you're after. If not, you may need to [create your own queries](https://github.com/olimorris/onedarkpro.nvim/blob/main/after/queries/rust/highlights.scm).

The `:Inspect` command will be your friend.
